docs/using-orbs.md

```diff
@@ -24,8 +31,7 @@ Orb URIs have the format:
 Orb namespaces may have restrictions that prevent you from accesing orbs in those namespaces based on whether the build and the scope of the namespace are congruent.
 
 ## Semantic Versioning in Orbs
-
-Orbs are published with the standard 3-number [semantic versioning system](https://semver.org/), `major.minor.patch`, and orb authors need to adhere to semantic versioning. Within `config.yml`, you can specify wildcard version ranges to resovle orbs. You can also use the special string `volatile` to pull in whatever the highest version number is.  
+Orbs are published with the standard 3-number [semantic versioning system](https://semver.org/), `major.minor.patch`, and orb authors need to adhere to semantic versioning. Within `config.yml`, you can specify wildcard version ranges to resolve orbs. You can also use the special string `volatile` to pull in whatever the highest version number is at time your build runs.  
```

Comment:
I think it's worth adding an example to this section to illustrate what `volatile` will not track.  I.e. when`node@8.0.0` exists, and `node@8.1.0` and `node@8.0.1` are published, `volatile` will refer to `node@8.1.0`. This is redundant with everything already in the PR, but specifics will help orb users think about such edge cases when writing their config.